.github/workflows/code_review.yml

```diff
@@ -12,9 +12,10 @@
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: ./
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4-1106-preview"
+          INFERENCE_API_KEY: ${{ secrets.INFERENCE_API_KEY }}
```

Comment:
The change from `OPENAI_API_KEY` to `INFERENCE_API_KEY` may indicate a shift in how the API is being accessed. Ensure that the new key is properly configured and that any dependencies or documentation are updated accordingly.